buildbot/repos/rpm/control/centos8/Minecraft-Overviewer.spec

```diff
@@ -0,0 +1,38 @@
+Summary: Generates large resolution images of a Minecraft map.
+Name: Minecraft-Overviewer
+Version: {VERSION}
+Release: 1%{?dist}
+Source0: %{name}-%{version}.tar.gz
+License: GNU General Public License v3
+Group: Development/Libraries
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
+Vendor: Andrew Brown <brownan@gmail.com>
+Url: http://overviewer.org/
+Requires: epel-release python3-pillow python3-numpy
+BuildRequires: epel-release git python36-devel python3-numpy
```

Comment:
somewhat sure this doesn't require epel-release at all

Reply:
doesn't appear to be required at build time, removed